### PR TITLE
Mobile sections

### PR DESCRIPTION
**Pull request by smunozmo**
# Template 5
[Figma Template #5
](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A1095)
## Updates:

- Changed tags for semantic tags.
- Created index.html and style.css for the first 2 sections.
- Added linters and .gitignore files.
- Updated README file.

No linters errors detected. ✅
Used flex-box for layout. ✅

### CODE REVIEW

**Review by Dipeshtwis — COMMENTED** [review]
# Status: Invalid ❌

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Reason

**Invalid Code Review Request**

You submitted a Pull Request (PR) that was already merged. Your PR should not be merged until a Code Reviewer approves it. You should use this [PR](https://github.com/smunozmo/Portfolio/pull/2) only for all the re-review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏


Cheers and Happy Coding :clap: :clap: :clap: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

**_Dipesh Kumar_**

**Comment by smunozmo** [comment]
> ## Reason
> **Invalid Code Review Request**
> 
> You submitted a Pull Request (PR) that was already merged. Your PR should not be merged until a Code Reviewer approves it. You should use this [PR](https://github.com/smunozmo/Portfolio/pull/2) only for all the re-review

Hi @Dipeshtwis,
Thanks for your review, there are two branches Main and Mobile-Sections, the Main branch has only the initial README file, it has not been merged.

-- Sebastián M.

**Comment by Dipeshtwis** [comment]
> > ## Reason
> > **Invalid Code Review Request**
> > You submitted a Pull Request (PR) that was already merged. Your PR should not be merged until a Code Reviewer approves it. You should use this [PR](https://github.com/smunozmo/Portfolio/pull/2) only for all the re-review
> 
> Hi @Dipeshtwis,
> Thanks for your review, there are two branches Main and Mobile-Sections, the Main branch has only the initial README file, it has not been merged.
> 
> -- Sebastián M.

Hi @smunozmo,
Please submit same PR link while requesting for code review for all the re-review. I can see that you have closed your previous PR, please reopen it. For your first review you have used [this PR](https://github.com/smunozmo/Portfolio/pull/2). Always use the same PR as next reviewer will review whatever changes you have done after that. Hope this will clears to you.

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+
+> Welcome to my personal portfolio, feel free to look around.
+
+
```
Comment (Shaher-11) [inline README.md:5]: Add the screenshot of the portfolio here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    </main>
+    <section class="intro">
+        <h2>
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. <br>
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </h2>
+    </section>
+    <div class="collab">
+        <button type="button">Start collaboration</button>
+    </div>
```
Comment (Shaher-11) [inline index.html:27]: Add margin between the button and paragrpah

File: index.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        </nav>
+    </header>
+    <main class="name">
+        <h1>Sebastián<br>Muñoz</h1>
+    </main>
+    <section class="intro">
+        <h2>
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. <br>
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </h2>
```
Comment (Shaher-11) [inline index.html:23]: Replace h2 with p as this is a paragraph and make sure the text is properly aligned on the page

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @smunozmo 

##### Great work on making the changes requested by the previous reviewer:clap::clap::clap:
You've done a wonderful job on the project so far:blush:
but there are still some which aren't done yet, 
so we need to implement those changes to get the project approved:muscle::muscle::muscle:

 ## Required Changes ♻️

- [ ] Check the comments for detailed feedback on the requested changes 
- [ ] Add a screenshot to the readme file

## Optional Suggestions 💡

_The following point is highly recommended to improve your code quality, Please consider implementing it

- Some spacing between the elements would make the project just like the original one:smiley:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

File: README.md (hunk 1 above)
Comment (smunozmo) [reply README.md:5]: Thank you Shaher for your review, do you mean my portfolio or a template screenshot?

Comment (smunozmo) [reply README.md:5]: Should I use my portfolio screenshot or Figma screenshot?

Comment (Dipeshtwis) [reply README.md:5]: Hi @smunozmo,
You should add your portfolio screenshot so any user can see your work through README file.

Comment (smunozmo) [reply README.md:5]: Ok great! Thank you.

**Pull request closed without merging**